components/schemes/Color/src/MDCSemanticColorScheme.h

```diff
@@ -80,6 +80,16 @@
 @property(nonnull, readonly, nonatomic) UIColor *onBackgroundColor;
 @end
 
+/**
+ An enum of default color schemes that are supported.
+ */
+typedef NS_ENUM(NSInteger, MDCColorSchemeDefaults) {
+  /**
+   The Material defaults, circa April 2018.
+   */
+  MDCColorSchemeDefaultsMaterial
```

Comment:
Will future iterations never be called Material or will this change over time?

Reply:
Was debating giving it a date, e.g. April2018. What do you think?

Reply:
If we're going to go with dates, then 2018April would keep things neat and tidy. Maybe 2018_04 or 201804?

Reply:
201804 sounds good to me. Will update.

Reply:
Done.